src/containers/trends/SMBGDaysContainer.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import SMBGDayPointsAnimated from '../../components/trends/smbg/SMBGDayPointsAnimated';
+import SMBGDayLineAnimated from '../../components/trends/smbg/SMBGDayLineAnimated';
+
+const SMBGDaysContainer = (props) => {
```

Comment:
Since we've agreed to use "date" language for dates and "day" language for day-of-week (Mon, Tues, etc.), could we change the name of this to `SMBGsPerDateContainer` or something similar?